README.md

```diff
@@ -1,2 +1,51 @@
 # Frontend
-This repo will be linked to Netlify for ease of deployment and code review
+## What does this project do?
+This is the frontend repository for our student forum application. This means the codebase will contain only React components and API calls to the backend server repo. If you want to contribute to the backend of our student forum, please request access to the "Backend" repo in the organisation if you dont have read access already.
+
+The frontend will be hooked to Netlify for continuous deployment on the main branch. This also means that all pull requests made will have a nice preview of the changes by viewing the Netlify link.
+
+## How do I get started?
+
+### Pre-requisites
+- install git: https://git-scm.com/downloads
+- install node and npm using node version manager (NVM): 
+#### Windows
+https://docs.microsoft.com/en-us/windows/dev-environment/javascript/nodejs-on-windows
+
+#### MacOS
+https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681
+
+#### Linux
+https://linuxize.com/post/how-to-install-node-js-on-ubuntu-18.04/
+
+### Contribute
+To get started to contributing, please create a new issue in the "Issues" tab describing the new bug or feature you want to add, and add it to the "Projects" Kanban board. 
+
+Afterwards: 
+
+  --> fork the repository 
+  
+  --> clone it on your computer 
+  
+  --> run ```npm install```
+  
+  --> create a new branch
+  
+  --> proceed to add your contributions
+  
+  --> create a pull request on the SE701-T5/Frontend repository from your forked repo. 
+  
+For more details, please read our CONTRIBUTING.md file.
```

Comment:
```suggestion
 run ```npm install```
```

It seems redundant to repeat what is already instructed in CONTRIBUTING.md